.travis.yml

```diff
@@ -1,20 +1,22 @@
 language: go
 sudo: false
 go:
-- 1.8
+- 1.8.1
 - 1.6.4
-- 1.7.4
+- 1.7.5
 install:
 - go get -t ./...
 - go get github.com/nats-io/gnatsd
 - go get github.com/mattn/goveralls
 - go get github.com/wadey/gocovmerge
-- go get honnef.co/go/tools/cmd/staticcheck
-- go get honnef.co/go/tools/cmd/gosimple
+- go get -u honnef.co/go/tools/cmd/staticcheck
+- go get -u honnef.co/go/tools/cmd/gosimple
+- go get -u github.com/client9/misspell/cmd/misspell
```

Comment:
Ok, just notice that misspell needs to be invoked with `.` not `./...` otherwise it is not going recursively into directories. It may then find issues in vendor, in which case we will have to explicitly specify all our directories, etc..